pkg/controller/controller_broker_test.go

```diff
@@ -464,7 +484,24 @@ func TestReconcileBrokerWithReconcileError(t *testing.T) {
 	}
 }
 
+// TestUpdateBrokerCondition ensures that with specific conditions
+// the broker correctly reflects the changes during updateBrokerCondition().
+//
+// The test cases are proving:
+// - broker transition from unset status to not ready results in status change new time
```

Comment:
as in I'm missing the "and" right?   results in status change and new transition time